views/apartments/index.erb

```diff
@@ -1,5 +1,5 @@
 <ul>
-  <li><a href='/apartments/1'>Apartment 1</a></li>
-  <li><a href='/apartments/2'>Apartment 2</a></li>
-  <li><a href='/apartments/3'>Apartment 3</a></li>
+<% @apartments.each do |apartment| %>
+  <li><%= apartment.address %></li>
```

Comment:
It'd be nice to have these be links to each apartment's show page, but this is a good start.